proxy-node-translator/src/dist/config.yml

```diff
@@ -16,7 +16,6 @@ proxyNodeResponseUrl: ${PROXY_NODE_RESPONSE_ENDPOINT}
 proxyNodeMetadataForConnectorNodeUrl: ${PROXY_NODE_METADATA_FOR_CONNECTOR_NODE_URL}
 
 hubUrl: ${HUB_URL}
```

Comment:
I don't think we need the hubUrl in the translator.

Reply:
There's a lot of cleanup to do in the Translator, including the config. We will do the cleanup in a separate story/PR

Reply:
Ok, sounds like a plan.